operations/configure_cray_cli.md

```diff
@@ -21,50 +25,195 @@ The `cray` CLI only needs to be initialized once per user on a node.
+   2021-12-21 15:50:52,714 - INFO    - ncn-s001: Success
+   2021-12-21 15:50:52,714 - INFO    - ncn-s002: Success
+   2021-12-21 15:50:52,714 - INFO    - ncn-s003: Success
+   2021-12-21 15:50:52,714 - INFO    - ncn-w001: Success
+   2021-12-21 15:50:52,714 - INFO    - ncn-w002: Success
+   2021-12-21 15:50:52,714 - INFO    - ncn-w003: Success
+   ```
 
-   If LDAP configuration was enabled, then use a valid account in LDAP instead of the example account 'vers'.
+   Now the `cray` CLI is operational on all nodes where success was reported. If a node was
+   unsuccessful with initialization, there will be an error reported. See the troubleshooting
+   section for additional information.
 
-   If LDAP configuration was not enabled, or is not working, then a Keycloak local account could be created.
-   See [Configure Keycloak Account](CSM_product_management/Configure_Keycloak_Account.md) to create this local account in Keycloak
-   and then use it instead of the example account 'vers'.
+1. Remove the temporary user after the install is complete.
 
+   When the install is complete and keycloak is fully populated with the correct end users,
+   call this script again with the cleanup option to remove the temporary user from keycloak
+   and uninitialize the `cray` CLI on all master and worker nodes in the cluster.
    ```bash
-   ncn# cray init
+   ncn# python3 /usr/share/doc/csm/install/scripts/craycli_init.py --cleanup
+   2021-12-21 15:52:31,611 - INFO    - Removing temporary user and uninitializaing the cray cli
+   2021-12-21 15:52:31,783 - INFO    - Deleted user 'craycli_tmp_user'
+   2021-12-21 15:52:31,798 - INFO    - Uninitializing nodes:
+   2021-12-21 15:52:32,714 - INFO    - ncn-m001: Success
+   2021-12-21 15:52:32,714 - INFO    - ncn-m002: Success
+   2021-12-21 15:52:32,714 - INFO    - ncn-m003: Success
+   2021-12-21 15:52:32,714 - INFO    - ncn-s001: Success
+   2021-12-21 15:52:32,714 - INFO    - ncn-s002: Success
+   2021-12-21 15:52:32,714 - INFO    - ncn-s003: Success
+   2021-12-21 15:52:32,714 - INFO    - ncn-w001: Success
+   2021-12-21 15:52:32,714 - INFO    - ncn-w002: Success
+   2021-12-21 15:52:32,714 - INFO    - ncn-w003: Success
    ```
 
-   When prompted, remember to use the correct username instead of 'vers'.
-   Expected output (including the typed input) should look similar to the following:
+   At this point the `cray` CLI will no longer be operational on these nodes until they are
+   initialized and authorized again with a valid keycloak user.
+
+   Optionally the `cray` CLI may be initialized with a valid keycloak user during the cleanup
+   operation so that it is left operational. To do this pass in a user and password with the
+   cleanup command:
+   ```bash
+   ncn# python3 /usr/share/doc/csm/install/scripts/craycli_init.py --cleanup -u MY_USERNAME -p MY_PASSWORD
+   2021-12-21 15:52:31,611 - INFO    - Removing temporary user and uninitializaing the cray cli
```

Comment:
**uninitializaing** should be **uninitializing**
Also, **cli** should be **CLI**
(needs to be fixed both in the source script and in the example output)